src/zjs_qemu_x86.json

```diff
@@ -4,8 +4,8 @@
     "zephyr_conf": {
         "qemu_x86": [
             "CONFIG_MAIN_STACK_SIZE=2048",
-            "CONFIG_RAM_SIZE=80",
-            "CONFIG_ROM_SIZE=256"
+            "CONFIG_RAM_SIZE=4096",
+            "CONFIG_ROM_SIZE=4092"
```

Comment:
4092 seems like an odd choice. :)

I don't know if it makes sense to go so far above real board specs though? Maybe 512 / 1024 like a higher end board?

Reply:
this is what they used in the latest Zephyr qemu sample, I found it weird too they use 4092, but it's what's working for them, so i just use that as is.